plutus-core/plutus-core/test/Pretty/Golden/Readable/Examples/Vec/churchVec.golden

```diff
@@ -1 +1,7 @@
-\a -> \(n :: (* -> *) -> * -> *) -> all (r :: ((* -> *) -> * -> *) -> *). (all (p :: (* -> *) -> * -> *). a -> r p -> r ((\(n :: (* -> *) -> * -> *) -> \(f :: * -> *) -> \z -> f (n f z)) p)) -> r (\(f :: * -> *) -> \z -> z) -> r n
\ No newline at end of file
+\a
+ (n :: (* -> *) -> * -> *) ->
+  all (r :: ((* -> *) -> * -> *) -> *).
+    (all (p :: (* -> *) -> * -> *).
+       a -> r p -> r
+                     ((\(n :: (* -> *) -> * -> *) (f :: * -> *) z -> f (n f z))
+                        p)) -> r (\(f :: * -> *) z -> z) -> r n
```

Comment:
I wonder if we should pretend UPLC/TPLC has let-expressions when rendering things in the readable mode.

Reply:
If you want to use PIR, use PIR :P

Reply:
Sorry, I now realize I didn't express myself properly: I _am_ going to introduce an option to print applied lambdas as `let`s, so that I can read the damn thing, and I wonder if it should be the default one :stuck_out_tongue: 

Plus, there's no PIR for UPLC anyway.